specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -88,6 +88,96 @@
         ]
       },
       "x-ms-parameter-location": "method"
+    },
+    "WeatherYear": {
+      "name": "year",
+      "description": "Year of the cyclone(s)",
+      "type": "integer",
+      "format": "int32",
+      "in": "query",
+      "x-ms-parameter-location": "method"
+    },
+    "WeatherBasinID": {
```

Comment:
We may need optional and required version for `WeatherBasinID` and `WeatherGovernmentID` like our search API does [here](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/maps/data-plane/Search/preview/1.0/search.json#L87)

Reply:
Yes. Thanks for sharing the reference.